### PR TITLE
Multisig supports 25 users / signatures via hardcoded array limit

### LINKED ISSUE #2: Supports 1-25 signatures
The wallet should be able to support an arbitrary number of signers meaning that we do not hardcode up to 25 variables in storage for the signers.

In this instance the goal is 1-25 signers.

### PR DESCRIPTION
**Pull request by Braqzen**
# Summary

## Type of change

- Improvement

## Changes

The following changes have been made:

- Replaced hardcoded limits from 2 to 25
- Reordered some fields
- Added CONTRIBUTING.md into root README.md

# Related Issues

Closes #2 

# Additional context

Will be replaced with vec once that is out.

### CODE REVIEW

**Comment by Braqzen** [comment]
> It may be more cumbersome to test the contract with a hardcoded value of 25 compared to say 5, but that's up to you.

Yeah, it will suck. When vec

File: multisig-wallet/src/abi.sw [hunk 1]
```diff
@@ -8,11 +8,11 @@ use data_structures::User;
 
 abi MultiSignatureWallet {
     fn constructor(users: [User;
-    2], threshold: u64);
+    25], threshold: u64);
```
[the remark this answers is not in the document]
Comment (Braqzen) [reply multisig-wallet/src/abi.sw:11]: Nope, you cannot do that with arrays. It errors

[the remark this answers is not in the document]
Comment (bitzoic) [reply multisig-wallet/src/abi.sw:11]: I cry

**Review by bitzoic — COMMENTED** [review]
wen vec

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**